skin/adminhtml/default/openmage/form.css

```diff
@@ -73,246 +68,216 @@ button.save, button.ok_button, button.add, .form-button.ok_button {
+      background: #888; }
+  button.fail {
+    background: #e63a3a;
+    color: #fff;
+    padding: 0 13px; }
+    button.fail:hover {
+      background: #fa4545; }
+  button.cancel_button {
+    background: #e63a3a;
+    color: #fff;
+    padding: 0 13px; }
+    button.cancel_button:hover {
+      background: #fa4545; }
+  button.icon-btn {
+    width: auto !important; }
+    button.icon-btn span {
+      text-indent: 0;
+      width: auto;
+      display: inline; }
+  button.large {
+    font-size: 22px;
+    padding: 14px 22px 16px; }
+  button.add-image span {
+    background: none;
+    padding: 0; }
+  button.add-widget span {
+    background: none;
+    padding: 0; }
+  button.go span {
+    background: none;
+    padding: 0; }
+  button.add-variable span {
+    background: none;
+    padding: 0; }
 
 .form-button:hover {
-  background: #888;
-}
+  background: #888; }
 .form-button.ok_button:hover {
-  background-color: #0090FF;
-}
+  background-color: #0090FF; }
 .form-button.large {
   font-size: 22px;
-  padding: 14px 22px 16px;
-}
+  padding: 14px 22px 16px; }
 
 .field-100 {
   background: none;
   border: 0;
-  padding: 0;
-}
+  padding: 0; }
 
 .form-list td.label label {
-  padding: 0 15px 0 0;
-}
+  padding: 0 15px 0 0; }
 .form-list td.scope-label {
   color: #999;
-  font-size: 11px;
-}
+  font-size: 11px; }
 .form-list td.value {
-  padding-left: 0 !important;
-}
-.form-list td.value input.input-text,
-.form-list td.value textarea,
-.form-list td.value select {
-  width: 330px;
-}
-.form-list td.value p.note {
-  color: #202856;
-  font-size: 11px;
-  font-weight: 400;
-}
+  padding-left: 0 !important; }
+  .form-list td.value input.input-text, .form-list td.value textarea, .form-list td.value select {
+    width: 330px; }
+  .form-list td.value p.note {
+    color: #202856;
+    font-size: 11px;
+    font-weight: 400; }
 
 .input-text.large {
   padding: 12px 12px 14px;
-  font-size: 18px;
-}
+  font-size: 18px; }
+
+td:has(input.swatch-option) {
+  display: flex;
+  gap: 2px; }
+  td:has(input.swatch-option) .swatch-delete {
+    cursor: pointer;
+    width: 14px;
+    height: 14px;
+    display: flex;
+    justify-content: center;
+    font-size: 10px;
+    background: #e63a3a;
+    color: #fff;
+    border-radius: 2px; }
+  td:has(input.swatch-option) .swatch-option {
+    width: 26px !important;
+    height: 26px; }
+
+td:has(input.swatch-disabled) {
+  background-image: url('data:image/svg+xml,<svg id="e5490f2e-3f54-4acd-9496-57eed667c240" data-name="Livello 1" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 28.41093 28.29249"><title>placeholder</title><rect x="1.20546" y="1.08375" width="26" height="26" fill="%23fff" stroke="%23ccc" stroke-miterlimit="10"/><line x1="0.70546" y1="27.58375" x2="27.70546" y2="0.70875" fill="none" stroke="%23ff003a" stroke-miterlimit="10" stroke-width="2"/></svg>');
+  background-repeat: no-repeat;
+  outline: 1px solid #c8c8c8;
+  border: 1px solid #fff;
```

Comment:
not a deal breaker but I kinda like it more without outline/border, is seems cleaner to me because it has many less lines, what do you think about that?

Reply:
yes, it's a bit hacky I know but I always had to do it for emulate a color picker empty

without border / outline
![whitout](https://github.com/OpenMage/magento-lts/assets/5071467/59d35d5b-b510-42f9-9070-a9f9b4dbea6c)

original
![original](https://github.com/OpenMage/magento-lts/assets/5071467/d9bcc44f-251e-4630-b950-f653e1ff17ed)

Reply:
understood